src/components/Home/header/Header.jsx

```diff
@@ -6,10 +6,14 @@ import { FaBars } from "react-icons/fa";
 import { AiOutlineSearch } from "react-icons/ai";
 import { MdNotifications, MdApps } from "react-icons/md";
 
-const Header = () => {
+const Header = ({ handleToggleSidebar }) => {
```

Comment:
This will be changed as a part of above comments

Reply:
changed prop name as per suggestion.